pyam/core.py

```diff
@@ -2314,7 +2314,7 @@ def concat(dfs, ignore_meta_conflict=False, **kwargs):
 
     Parameters
     ----------
-    dfs : list of IamDataFrames
+    dfs : iterable of IamDataFrames
         A list of :class:`IamDataFrame` instances
```

Comment:
Any object castable as IamDataFrame to be appended
```suggestion
        A list of objects castable to :class:`IamDataFrame`
```